featuretools/entityset/deserialize.py

```diff
@@ -4,8 +4,8 @@
 import tempfile
 
 import pandas as pd
-import woodwork as ww
 from woodwork.deserialize import read_woodwork_table
+from woodwork.type_sys.type_system import type_system
```

Comment:
What do you think about importing this as `ww_type_system` to make it a little more clear when reading the code on line 73 that this is a woodwork method call?

Reply:
Sure, I don't mind updating